packages/app-mobile/components/screens/Note.tsx

```diff
@@ -1206,6 +1391,61 @@ class NoteScreenComponent extends BaseScreenComponent {
 
 		const noteTagDialog = !this.state.noteTagDialogShown ? null : <NoteTagsDialog onCloseRequested={this.noteTagDialog_closeRequested} />;
 
+		// Different styles for when the note actions are active or not
+		const notesBarToggleIconStyle = this.props.showNotesBar ? this.styles().noteActionButtonIconActive : this.styles().noteActionButtonIcon;
+		const notesBarToggleStyle = this.props.showNotesBar ? this.styles().noteActionButtonActive : this.styles().noteActionButton;
+
+		const handleNoteActionsDrag = (gestureState: any) => {
+			const minY = 160;
+			const maxY = 0.8 * Dimensions.get('window').height;
+
+			let newY = gestureState.moveY;
+
+			if (newY < minY) {
+				newY = minY;
+			} else if (newY > maxY) {
+				newY = maxY;
+			}
+
+			this.noteActionsPositionY.setValue(newY - 162);
+		};
+
+		// Pan responder that handles making the note actions draggable.
+		// The note actions need to be draggable, because they could
+		// potentially obstruct some portion of a note's content
+		this.noteActionsDragResponder = PanResponder.create({
```

Comment:
Why is this `this.noteActionsDragResponder` and not `const noteActionsDragResponder =`

Reply:
I don't remember why I did that, thanks for spotting it, changing it to const.